observability/network_observability/netobserv_cli/netobserv-cli-install.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="netobserv-cli-install"]
+= Installing the Network Observability CLI
+include::_attributes/common-attributes.adoc[]
+:context: cli-install
+
+toc::[]
+
+The Network Observability CLI (`oc netobserv`) is deployed separately from the Network Observability Operator. The CLI is available as an :oc-first: plugin. It provides a lightweight way to quickly debug and troubleshoot with Network Observability.
```

Comment:
This is a good question because I was just going off of the Operator capitalization, but you're right to point out that this isn't the Operator I'm talking about here, its the idea of network observability. I will need to dig more into this to find out.

Reply:
So for non-operator or cli references, I think the lowercase is the way to go.